### ISSUE TITLE
Allowing authors of legacy projects to release new versions

### ISSUE THREAD

**Issue by tompollard** [MIT-LCP/physionet-build#543]
Not really an issue, but something that I thought it would be good to record here. We've had a request from an author of a legacy project to create an updated version (amend files). The following process seems safe, from testing on staging:

1. Ensure the user is registered on the new platform.
2. Add the user as the submitting author of the legacy project using the admin tools (update the PUBLISHED AUTHORS field)
3. To submit the update, it may be necessary to complete required fields (e.g. if the legacy project doesn't have a DOI, a DOI should be added).
4. User is then able to create a new version in the usual way, by selecting the project from their project home (https://alpha.physionet.org/projects/).

If there's anything else that needs thinking about when creating an updated version of a legacy project, please comment here...

**Comment by bemoody** [comment]
We need to add authors to legacy projects anyway (see also issue
#386).  But it would be difficult to tie those to new user accounts
automatically - we could try to associate them by email address when
we know their email address, but may be more work than it's worth.

Can any author create a new version of a project?

When an author creates a new version, are the other authors then
automatically invited?

**Comment by elfeto** [comment]
How about, we add a button near the authorship in which creates a request of a logged user, to try and claim authorship, in which we have to decide ? (might be annoying if people want to spam, and it can be a bit informal)

On new versions the authors are automatically re-created without notice.

**Comment by tompollard** [comment]
> But it would be difficult to tie those to new user accounts automatically - we could try to associate them by email address when we know their email address, but may be more work than it's worth.

I agree it doesn't seem worth the effort spending development time on linking authors to legacy projects. It's a task that's fairly easy to deal with manually.

> Can any author create a new version of a project?

Only the submitting author of the original project is able to create a new version.

> When an author creates a new version, are the other authors then automatically invited?

The original authors are automatically included in the new version, but the author list can then be edited by the submitting author (for example, the order can be changed, authors can be removed, new authors can be added).
